tests/Mf2/ClassicMicroformatsTest.php

```diff
@@ -238,4 +238,444 @@ public function testRelBookmarkUrl() {
 
 	}
 
+
+	/**
+	 * @see http://microformats.org/wiki/microformats2-parsing-issues#uf2_children_on_backcompat_properties
+	 */
+	public function testMf2ChildrenOnBackcompatProperties()
+	{
+		$input = '<div class="vcard">
+  <div class="adr h-custom">
```

Comment:
Change `h-custom` to `h-acme-some-acme-object` to clarify that it's a non-standard property.